package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-farmer-apply",
-  "version": "0.55.3",
+  "version": "0.55.5",
```

Comment:
I think perhaps you still have a conflict here?
You could try merging main to resolve it.